x-pack/plugins/security_solution/common/utils/risk_score_modules.test.ts

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RiskScoreEntity } from '../search_strategy';
+import {
+  getCreateLatestTransformOptions,
+  getCreateMLHostPivotTransformOptions,
+  getCreateMLUserPivotTransformOptions,
+  getCreateRiskScoreIndicesOptions,
+  getCreateRiskScoreLatestIndicesOptions,
+  getIngestPipelineName,
+  getLatestTransformIndex,
+  getPivotTransformIndex,
+  getRiskScoreIngestPipelineOptions,
+  getRiskScorePivotTransformId,
+  getRiskHostCreateInitScriptOptions,
+  getRiskHostCreateLevelScriptOptions,
+  getRiskHostCreateMapScriptOptions,
+  getRiskHostCreateReduceScriptOptions,
+  getRiskScoreInitScriptId,
+  getRiskScoreLevelScriptId,
+  getRiskScoreMapScriptId,
+  getRiskScoreReduceScriptId,
+  getRiskUserCreateLevelScriptOptions,
+  getRiskUserCreateMapScriptOptions,
+  getRiskUserCreateReduceScriptOptions,
+  getLegacyIngestPipelineName,
+  getLegacyRiskScoreLevelScriptId,
+  getLegacyRiskScoreInitScriptId,
+  getLegacyRiskScoreMapScriptId,
+  getLegacyRiskScoreReduceScriptId,
+} from './risk_score_modules';
+
+const mockSpaceId = 'customSpaceId';
+describe('getRiskScorePivotTransformId', () => {
```

Comment:
I sent you the file over slack, but lets consolidate these with a forEach function `[RiskScoreEntity.host, RiskScoreEntity.user].forEach(entity => runTests(entity))`